platform/formats/i18next_json.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: i18next_json
+title: i18next JSON
+sidebar_label: i18next JSON
+---
+
+i18next is a library and format commonly used for handling translations in JavaScript-based projects. Translations are usually stored as part of JavaScript code or in `.json` files. Tolgee supports importing the `.json` representation of translation mapping, but not all i18next features are supported.
+
+The i18next format uses suffixes for plurals (e.g. `my_key_zero`, `my_key_one`, `my_key_many`) and has specific placeholders for keys (e.g. `value is {{placeholder_key}}`). You can import and export your localization data in the i18next format directly using Tolgee.
+
+However, when possible, we recommend using the **ICU format** with the **[ICU plugin](https://github.com/i18next/i18next-icu) for i18next**. The ICU format offers much better support for advanced message formatting and plurals.
+
+Read more about i18next in [i18next official documentation](https://www.i18next.com/).
+
+
+## Example of i18next JSON
+
+```json title="en.json"
+{
+  "save": {
+    "button": "Save"
+  },
+  "dogs_count_zero": "No dogs",
+  "dogs_count_one": "One dog",
+  "dogs_count_other": "{{count}} dogs",
+  "array": [
+    "First item",
+    "Second item"
+  ]
+}
+```
+
+When importing this JSON, Tolgee will convert nested keys into flat key structure and all plurals and placeholders to ICU message format (can be disabled).
+
+When exporting to i18next JSON, Tolgee will convert ICU message format back to i18next format. Depending on your preference, you can choose to either use the **flat JSON structure** or **nested JSON structure**. However, for projects needing more advanced message formatting, we strongly recommend using the ICU format with the i18next [ICU plugin](https://github.com/i18next/i18next-icu). The Tolgee platform doesn't support exporting these advanced features in the i18next format.
+
+
+## The message format and placeholder conversion
+
+import {
+  GenericMessageAndPlaceholderConversion
+} from "../../src/component/docs/formats/GenericMessageAndPlaceholderConversion";
+
+<GenericMessageAndPlaceholderConversion formatName="i18next JSON"
+                                        placeholderExamples={<>
+                                          <code>{'{{'}name{'}}'}</code>, <code>{'{{'}count, number{'}}'}</code></>} />
+
+### Placeholder conversion specification
+
+Only specifiers specified in the table below are supported. Tolgee doesn't support additional flags and modifiers and won't convert such placeholders.
```

Comment:
You are not mentioning how it behaves when unsupported placeholder is imported and exported. We need to asure users that when they import some data, the same data are exported. That's important.

Reply:
True. I've added something to explain it better. Hopefully, it will be understandable to users.